tasks/vagrant.py

```diff
@@ -211,8 +211,10 @@ def clean_last_time_used(box):
 
         # Do not delete Windows boxes
         linux_boxes = [x for x in all_boxes if 'windows' not in x.name]
-        if len(linux_boxes) > 4:
-            linux_boxes[0].delete_box()
+
+        # Keep only the latest 3 boxes
+        for box in linux_boxes[:-3]:
```

Comment:
@f-trivino this is executed when the runner wants to download a new template, so we can keep only that template that will be downloaded and delete the ones in the runner with:
```python
for box in linux_boxes:
    box.delete_box()
```
The problem was that some runners were keeping 6 boxes (I don't know why), they are larger, we were keeping 5 and previous code was only deleting one on them.